### PR TITLE
Megafauna won't heal while on station

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Megafauna will not heal while on the station. Do not be afraid to throw your life away to get in a few toolbox hits.
/:cl:

I thought megafauna rampages were more fun when they killed like 40 people but the survivors inevitably wore it down with heroic suicide attacks. This should make them both more manageable and more exciting, rather than having infinite health from dumb assistants and simple animals.

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -734,6 +734,10 @@
 		user << "[choice] is already dead!"
 		used = FALSE
 		return
+	if(choice == user)
+		user << "You feel like writing your own name into a cursed death warrant would be unwise."
+		used + FALSE
+		return
```
Comment (XDTM) [inline code/modules/mining/lavaland/necropolis_chests.dm]: forgot to swap branches?

Comment (KorPhaeron) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Yeah, oh well

**Comment by ChangelingRain** [comment]
This won't work for the Hierophant but frankly that doesn't matter because the Hierophant isn't going to stick around for very long.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -734,6 +734,10 @@
 		user << "[choice] is already dead!"
 		used = FALSE
 		return
+	if(choice == user)
+		user << "You feel like writing your own name into a cursed death warrant would be unwise."
+		used = FALSE
+		return
```
Comment (Cyberboss) [inline code/modules/mining/lavaland/necropolis_chests.dm]: MERGING PRS FOR SHAME!

**Comment by Kromgar** [comment]
Does this effect lesser ash drakes?

**Comment by Kromgar** [comment]
>stealthily merging pr to make blood contract unusable on self

Reeeeeeeeee

**Comment by Militaires** [comment]
@Kromgar yes.

**Comment by KorPhaeron** [comment]
@Kromgar 

Player controlled is exempt, so no

**Comment by Kromgar** [comment]
Im totally okay with this then

**Comment by darkpaladin109** [comment]
Are they still gonna gib/disintegrate people?

**Comment by Kromgar** [comment]
@darkpaladin109 Once a megafauna is on the station its a shuttle call. The stations gonna get wrecked

**Pull request merged**